packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -661,15 +682,22 @@ export class StatsAnalyzer extends EventsScope {
    */
   private compareLastStatsResult() {
     if (this.lastStatsResults !== null && this.meetingMediaStatus) {
-      const getCurrentStatsTotals = (keyPrefix: string, value: string): number =>
+      const getCurrentStatsTotals = (keyPrefix: string, value: string, send: boolean): number =>
         Object.keys(this.statsResults)
           .filter((key) => key.startsWith(keyPrefix))
-          .reduce((prev, cur) => prev + (this.statsResults[cur]?.recv[value] || 0), 0);
+          .reduce(
+            (prev, cur) => prev + (this.statsResults[cur]?.[send ? 'send' : 'recv'][value] || 0),
+            0
+          );
```

Comment:
Also was this just straight up incorrect logic previously? (Just want to make sure I understand the purpose of this change.)

Reply:
This change was so that the function could aggregate sender values when previously it was only used for aggregating receiver values.